src/auth/auth.service.ts

```
@@ -15,58 +16,23 @@ import * as bcrypt from 'bcryptjs';
 import type { RegisterDto } from './dto/register-user.dto';
 import type { CreatePortfolioDto } from './dto/create-portfolio.dto';
 import { JwtService } from '@nestjs/jwt';
-import { addHours } from 'date-fns';
+import { addHours, addDays, addMinutes } from 'date-fns';
```

ESLint

[error] 'addHours' is defined but never used.

(@typescript-eslint/no-unused-vars)